raw_data/preprocessed/africa/lesotho_english.txt

```diff
@@ -0,0 +1,379 @@
llowing purposes: [a] requiring the person whose activities have degrading effect on the environment or natural resources to restore that environment or natural resources as near as it may be to the state in which it was before he began activities which have degrading effect on the environment or natural resources; [b] preventing the person from taking an action which is causing or is likely to cause harm to the environment or natural resources. [3] An environmental restoration notice may have any terms or conditions and impose on the person on whom it is served, obligations, which in the opinion of the Director may enable the notice to achieve the purposes for which it was issued. [4] Notwithstanding the provisions of this section, the restoration notice may require a person on whom it is served to: [a] take an action, which may prevent the commencement or continuation or the cause of pollution; [b] restore land, including the replacement of soil, the replanting of trees and other flora and outstanding geological, archaeological or historical features of the land or the area to the land specified in the order; [c] take an action which shall prevent the commencement or continuation or cause of the environmental hazard; [d] cease to take an action, which is causing or is likely to cause pollution or an environment hazard; [e] remove or alleviate damage to land or the environment or to the amenities of the areas; [f] prevent damage to the land or the environment, aquifers beneath the land and flora and fauna in, on or under the land specified in the order or land or the environment contiguous to the land specified in the order; or [g] remove or dispose of waste or refuse deposited on land specified in the notice. [5] The Director shall have power to inspect, at any reasonable time, any activity on any premises for purposes of determining whether the activity is harmful to the environment of the conservation of natural resources or whether to make an environmental restoration notice under subsection [1]. [6] For purposes of subsection [5], the Director may enter any premises at any reasonable time to enforce the environmental restoration notice and the Director shall not be responsible for the consequences of any action reasonably taken by him in good faith under this section. [7] The Director my delegate the power referred to in subsections [5] and [6] to an inspector or any person duly authorised by him and anything done by that person shall be deemed to be done by the inspector or by the Director and shall be valid for all purposes. [8] In exercising the powers under this section, the Director shall be guided by the principles of good environmental management specified in this Act. [9] A person who fails or refuses to comply with the environmental restoration notice, made under this section commits an offence and is liable on conviction to imprisonment for a fine not less than M5,000 or to imprisonment for a term not less than 2 years or to both.
```

Comment:
Please replace `the Director my delegate` with `the Directory may delegate`. Also, please replace `restoration notice, made` with `restoration notice made`.